http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200515_add_runtime_disk_foreign_key.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="rtitle" id="add_runtime_disk_foreign_key">
+        <addColumn tableName="CLUSTER">
+            <column name="persistentDiskId" type="BIGINT">
+                <constraints nullable="true" />
```

Comment:
I made it a nullable foreign key, because existing runtime records in the DB won't have a disk record. Nullable foreign keys are OK in MySQL.